sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/Sample.java

```diff
@@ -202,10 +202,10 @@ public void populateDisplayData(DisplayData.Builder builder) {
   }
 
   /** A {@link CombineFn} that combines into a {@link List} of up to limit elements. */
-  private static class SampleAnyCombineFn<T> extends CombineFn<T, List<T>, Iterable<T>> {
+  public static class SampleAnyCombineFn<T> extends CombineFn<T, List<T>, Iterable<T>> {
```

Comment:
nit: doesn't look necessary, we can probably replicate this combiner in tests, instead of extending public API

Reply:
Sample.anyCombineFn() exists, so I reverted this change